EIPS/eip-X-identity.md

```diff
@@ -0,0 +1,302 @@
+---
+eip: <to be assigned>
+title: ERC-725 Identity
+author: Fabian Vogelsteller (@frozeman) <fabian@ethereum.org>
```

Comment:
I don't think the build script will know how to parse this into a single link.
Either GitHub username **or** e-mail address should be sufficient.
See: https://github.com/ethereum/EIPs/blob/master/_includes/authorlist.html